packages/ai-chat/src/common/chat-session-summary-agent-prompt.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/tslint/config  */
+// *****************************************************************************
+// Copyright (C) 2025 EclipseSource GmbH and others.
+//
+// This file is licensed under the MIT License.
+// See LICENSE-MIT.txt in the project root for license information.
+// https://opensource.org/license/mit.
+//
+// SPDX-License-Identifier: MIT
+
+import { CHANGE_SET_SUMMARY_VARIABLE_ID } from './context-variables';
+
+export const CHAT_SESSION_SUMMARY_PROMPT = {
+    id: 'chat-session-summary-prompt',
+    template: `{{!-- Made improvements or adaptations to this prompt template? We\'d love for you to share it with the community! Contribute back here: ' +
```

Comment:
```suggestion
    template: `{{!-- !-- This prompt is licensed under the MIT License (https://opensource.org/license/mit).
Made improvements or adaptations to this prompt template? We\'d love for you to share it with the community! Contribute back here: ' +
```

Reply:
It looks like we haven't done this - either the move to an MIT-licensed licensed or the addition of the MIT license notice to the prompt - with the prompt for chat session naming, which is what I based this one off of. Do we want to do either of those things?